output_template/python/config.py

```diff
@@ -15,64 +15,137 @@
-            for k in class_args:
-                v = class_args[k]
-                processor.__dict__[k] = v
-            processors.append(processor)
-    seq = Sequence(processors=processors)
-    return seq
+    """The pre processor is loaded based on the information passed,
+    It is append to a Sequence object and returned in.
+
+    Args:
+        pre_processor_config (List[Dict[str, Optional[Dict[str, Any]]]]): List of processors to load.
+
+    Returns:
+        Sequence: A Sequence object with a list of processors inside.
+
+    Examples:
+        >>> config = [
+                {"Resize": {"size": [128, 128], "resample": "NEAREST"}},
+                {"PerImageStandardization": None},
+            ]
+        >>> pre_processors = build_pre_process(config)
+    """
+    return _build_process(pre_processor, pre_processor_config)
 
 
 def build_post_process(post_processor_config):
-    module_name = "blueoil/post_processor"
-    f, pathname, description = imp.find_module(module_name)
-    module = imp.load_module(module_name, f, pathname, description)
+    """The post processor is loaded based on the information passed,
+    It is append to a Sequence object and returned in.
+
+    Args:
+        post_processor_config (List[Dict[str, Optional[Dict[str, Any]]]]): List of processors to load.
+
+    Returns:
+        Sequence: A Sequence object with a list of processors inside.
+
+    Examples:
+        >>> config = [
+                {"FormatYoloV2": {
+                    "anchors": [
+                        [1.3221, 1.73145],
+                        [3.19275, 4.00944],
+                        [5.05587, 8.09892],
+                        [9.47112, 4.84053],
+                        [11.2364, 10.0071],
+                    ],
+                    "boxes_per_cell": 5,
+                    "data_format": "NHWC",
+                    "image_size": [128, 128],
+                    "num_classes": 1,
+                }},
+                {"ExcludeLowScoreBox": {"threshold": 0.3}},
+                {"NMS": {
+                    "classes": ["person"],
+                    "iou_threshold": 0.5,
+                    "max_output_size": 100,
+                    "per_class": True,
+                }},
+            ]
+        >>> post_processors = build_post_process(config)
+    """
+    return _build_process(post_processor, post_processor_config)
+
+
+def _build_process(module, processor_config=None):
     processors = []
-    if post_processor_config is None:
-        post_processor_config = {}
-    for p in post_processor_config:
-        for class_name in p:
-            class_args = p[class_name]
-            if class_args is None:
-                class_args = {}
-            cls = getattr(module, class_name)
-            # Create none initialized processor `cls` instance.
+    processor_config = processor_config or []
+    for p in processor_config:
+        for class_name, kwargs in p.items():
+            try:
```

Comment:
I think it's better to use `if` `else` than `try` `catch`.
Can you rewrite like

```python
if hasattr(module, class_name):
    cls = getattr(module, class_name)
else:
    cls = import_from_string(class_name)
```

Sorry for not documented coding style...